<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReclassifyDialog</class>
 <widget class="QDialog" name="ReclassifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reclassify Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabReclassification">
      <attribute name="title">
       <string>Class Mapping</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,2,0">
       <item row="2" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Source Class</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Target Class</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnSelectSrcfile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source File</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="mapLayerComboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTargetClassification">
      <attribute name="title">
       <string>Target Classification</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,1">
       <item row="0" column="3">
        <widget class="QPushButton" name="btnSelectDstFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="tbDstFile"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QgsRasterFormatSaveOptionsWidget" name="widgetDstFile" native="true">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="ClassificationSchemeWidget" name="dstClassificationSchemeWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClassificationSchemeWidget</class>
   <extends>QWidget</extends>
   <header>enmapbox.gui.classificationscheme</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsRasterFormatSaveOptionsWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReclassifyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReclassifyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
